back/app/services/project_copy_service.rb

```diff
@@ -531,6 +542,18 @@ def yml_event_files(shift_timestamps: 0)
     end
   end
 
+  def yml_event_images(shift_timestamps: 0)
+    @project.events.flat_map(&:event_images).map do |image|
+      {
+        'event_ref' => lookup_ref(image.event_id, :event),
+        'remote_image_url' => image.image_url,
```

Comment:
Assume in copying, the copied event will just reference the same image on S3?

Reply:
When copying, the images are also copied over, but the template itself references to the original image.